humilis_batch/__init__.py

```diff
@@ -1,5 +1,5 @@
 """Humilis plug-in to deploy AWS Batch resources."""
 
 
-__version__ = "0.5.1"
+__version__ = "0.6.1"
```

Comment:
Patch version must be reset to 0 when minor version is incremented.

```suggestion
__version__ = "0.6.0"
```

Patch version MUST be reset to 0 when minor version is incremented.